devtools-ci-index.yaml

```diff
@@ -3132,7 +3132,7 @@
             timeout: 60m
             scenario: launch-booster
             server_address: https://openshift.io
-            oso_cluster_address: https://api.starter-us-east-2.openshift.com:443
+            oso_cluster_address: https://api.starter-us-east-2a.openshift.com:443
```

Comment:
few lines above you have`cluster: us-east-2` so I wonder is it also should be updated? because it looks inconsistent